integration-tests/integration/appointments/addAppointment.spec.js

```diff
@@ -0,0 +1,311 @@
+          eventLocation: 'Visiting room',
+          eventDescription: 'Visits',
+          comment: 'Friends',
+          startTime: '2017-10-15T18:00:00',
+          endTime: '2017-10-15T18:30:00',
+          locationId: 3,
+        },
+      ],
+      activities: [
+        {
+          offenderNo,
+          firstName: 'TEST',
+          lastName: 'USER',
+          cellLocation: 'LEI-A-1-1',
+          comment: 'Appt details',
+          event: 'MEDE',
+          eventId: 106,
+          eventDescription: 'Medical - Dentist',
+          eventLocation: 'Medical Room1',
+          startTime: '2017-10-15T15:30:00',
+          locationId: 4,
+        },
+      ],
+    })
+    cy.task('stubSentenceData')
+    cy.task('stubLocation', 1)
+    cy.task('stubCourts')
+    cy.task('stubAddVideoLinkAppointment')
+    cy.task('stubAgencyDetails', 'MDI')
+    cy.task('stubUserEmail', 'ITAG_USER')
+
+    cy.visit(`/offenders/${offenderNo}/add-appointment`)
+  })
+
+  it('A user can successfully add an appointment', () => {
+    const addAppointmentPage = AddAppointmentPage.verifyOnPage()
+    const form = addAppointmentPage.form()
+    form.date().type(moment().format('DD/MM/yyyy'))
+    form.appointmentType().select('ACTI')
+    form.location().select('1')
+    form.startTimeHours().select('23')
+    form.startTimeMinutes().select('55')
+    addAppointmentPage.offenderEvents().contains('Visiting room - Visits - Friends')
+    addAppointmentPage.locationEvents().contains('Medical Room1 - Medical - Dentist - Appt details')
+    form.recurringNo().click()
+    form.comments().type('Test comment')
+    form.submitButton().click()
+    ConfirmSingleAppointmentPage.verifyOnPage()
+  })
+
+  it('Schedules remain after validation error', () => {
+    const addAppointmentPage = AddAppointmentPage.verifyOnPage()
+    const form = addAppointmentPage.form()
+    form.date().type(moment().format('DD/MM/yyyy'))
+    form.appointmentType().select('ACTI')
+    form.location().select('1')
+    addAppointmentPage.offenderEvents().contains('Visiting room - Visits - Friends')
+    addAppointmentPage.locationEvents().contains('Medical Room1 - Medical - Dentist - Appt details')
+    form.recurringNo().click()
+    form.comments().type('Test comment')
+    form.submitButton().click()
+    addAppointmentPage.errorSummary().contains('Select a start time')
+    addAppointmentPage.offenderEvents().contains('Visiting room - Visits - Friends')
+  })
+
+  it('A user can successfully add a recurring appointment', () => {
+    const addAppointmentPage = AddAppointmentPage.verifyOnPage()
+    const form = addAppointmentPage.form()
+    form.appointmentType().select('ACTI')
+    form.location().select('1')
+    form.startTimeHours().select('23')
+    form.startTimeMinutes().select('55')
+    form.date().type(moment().format('DD/MM/yyyy'))
+    form.recurringYes().click()
+    form.repeats().select('DAILY')
+
+    addAppointmentPage.lastAppointmentDate().should('not.be.visible')
```

Comment:
This validates the "Last appointment" section is hidden until the times to repeat have been entered.